jerry-core/ecma/builtin-objects/ecma-builtin-global.cpp

```diff
@@ -631,7 +636,7 @@ ecma_builtin_global_object_decode_uri_helper (ecma_value_t uri __attr_unused___,
         output_size++;
       }
     }
-    else if (decoded_byte < 0xc0 || decoded_byte >= 0xf8)
+    else if (decoded_byte < LIT_UTF8_2_BYTE_MARKER || decoded_byte >= 0xf8)
```

Comment:
Maybe, it is better to add `LIT_UTF8_5_BYTE_MARKER` as a specification-defined value, and also `#define LIT_UTF8_FIRST_BYTE_MAX LIT_UTF8_5_BYTE_MARKER` with comment, saying that it is not a specification-defined value, but maximum possible value in the whole engine.